Sources/TuistKit/Services/GenerateService.swift

```diff
@@ -0,0 +1,45 @@
+import TuistGenerator
+import TuistLoader
+import TuistSupport
+import Basic
+
+protocol GenerateServicing {
```

Comment:
The `GenerateCommand ` executes the following under the hood of `run` method:

```swift
func run() throws {
  try GenerateService().run(
    path: path,
    projectOnly: projectOnly
  )
}
```
According to what [Pedro suggested](https://github.com/tuist/tuist/issues/1040#issuecomment-604259949), the logic of the command will be extracted from the command to test the logic. The command will be no more than a simple shell for parsing the arguments. 

Taking all of that into account, I've been wondering if we really need to introduce an extra protocol `GenerateServicing`?  Isn't that just a boilerplate in our case? 🤔

Reply:
That's a good point, I suppose having protocol has no added benefit here since we will only use the instance (similarly, commands in SPM do not have a protocol since unnecessary).

I think we can delete this 👍